src/components/Form.js

```diff
@@ -23,31 +24,31 @@ function Form() {
     setInput(input);
   };
   return (
-    <form>
-      <label htmlFor="label">
-        ADD NEW BOOK
-        <div>
-          <input
-            type="text"
-            onChange={getInputs}
-            name="title"
-            value={input.title}
-            placeholder="Book Title"
-          />
-          <br />
-          <input
-            type="text"
-            name="author"
-            onChange={getInputs}
-            value={input.author}
-            placeholder="Author"
-          />
-          <br />
-          <button type="submit" onClick={handleSubmit}>
-            ADD BOOK
-          </button>
-        </div>
-      </label>
+    <form className="book-form">
+      <h2 className="add-n-book">ADD NEW BOOK</h2>
```

Comment:
- [ ] Kindly use the correct font family for the `ADD NEW BOOK` heading as well as per requirement.